### PR TITLE
PodDisruptionBudget for VM, vmagent and vmalert

### PR DESCRIPTION
**Pull request by chrissng**
Allow enabling PodDisruptionBudget for:
- VM cluster (vmselect, vmstorage and vminsert)
- vmagent
- vmalert

### CODE REVIEW

File: charts/victoria-metrics-agent/templates/pdb.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{{- if .Values.podDisruptionBudget.enabled }}
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ include "chart.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "chart.labels" . | nindent 4 }}
+  {{- if .Values.podDisruptionBudget.labels }}
```
Comment (tenmozes) [inline charts/victoria-metrics-agent/templates/pdb.yaml:9]: could you please use {{ with }}, i think this is more readable

Comment (chrissng) [reply charts/victoria-metrics-agent/templates/pdb.yaml:9]: Sure i'll do that.

Comment (chrissng) [reply charts/victoria-metrics-agent/templates/pdb.yaml]: Fixed

**Comment by tenmozes** [comment]
@chrissng thank you for contribution

**Pull request merged**